_docs/concepts/what-is-kritis/overview.md

```diff
@@ -115,6 +115,9 @@ As an exception to this rule, our policy language allows glob-based container im
 ## Conclusion and Next Steps
 Binary Authorization enables centralized control over software release cycle. Stakeholders configure policies to enforce the requirements of the release process, gaining confidence that software is delivered to customers only if it meets the organization’s requirements. Attestations - trusted metadata associated to software artifacts - are used to assert that software meets specified requirements.
 
+Kritis Beta release is availabe [here](https://github.com/grafeas/kritis/releases). For more information and tutorial, please check out the [Kritis tutorial](https://github.com/grafeas/kritis/blob/master/tutorial.md).
```

Comment:
Could this be worded to avoid saying tutorial twice within a sentence? How about:

"For more information, please see the [tutorial]". 

Also, spelling of 'available'.